assets/pages/bundles-landing/support-landing-ab-test/supportLanding.scss

```diff
@@ -240,18 +241,18 @@
     .component-cta-link {
       height: 54px;
       line-height: 54px;
-      background-color: gu-colour(comment-main-2);
+      background-color: gu-colour(multimedia-main-2);
       color: gu-colour(neutral-1);
       font-size: 14px;
       font-weight: bold;
       padding-left: 20px;
       padding-top: 0;
       padding-bottom: 0;
       box-sizing: border-box;
-      margin-top: $gu-v-spacing * 2;
+      margin-top: $gu-v-spacing;
 
       &:hover {
-        background-color: darken(gu-colour(comment-main-2), 5%);
+        background-color: gu-colour(light-yellow);
```

Comment:
[For a separate PR] Since we are starting a new design soon, it would be useful to invest time in improving our naming system.